cmake/CPM.cmake

```diff
@@ -0,0 +1,24 @@
+# SPDX-License-Identifier: MIT
+#
+# SPDX-FileCopyrightText: Copyright (c) 2019-2023 Lars Melchior and contributors
```

Comment:
And if you wanted to include copyright as well, consider including this type of line as well